### PR TITLE
Add new OMs of the Month

### PR DESCRIPTION
**Pull request by Pikachuun**
~~Mix and Mega is most likely going to be temporary until Partners in Crime is coded.~~
Nevermind, Mix and Mega is now officially OM of the Month, so it's permanent.

Done by request of @TheImmortal.

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -366,63 +366,147 @@ exports.Formats = [
...
+			}
+			if (item.id === 'beedrillite' || item.id === 'kangaskhanite') {
+				return ["" + item.name + " can only allowed be held by " + item.megaEvolves + "."];
+			}
+			switch (item.id) {
+			case 'blazikenite':
+				if (set.ability !== 'Speed Boost') return ["" + template.species + " is not allowed to hold " + item.name + "."];
+				break;
+			case 'mawilite': case 'medichamite':
+				let powerAbilities = {'Huge Power':1, 'Pure Power':1};
```
Comment (Slayer95) [inline config/formats.js]: This case needs to be block-scoped, Or it will crash PS main

File: config/formats.js [hunk 2]
```diff
@@ -366,63 +366,147 @@ exports.Formats = [
...
+				pokemon.originalSpecies = pokemon.baseTemplate.species;
 			}
 		},
+		onSwitchInPriority: -6,
 		onSwitchIn: function (pokemon) {
-			if (pokemon.side.god.hp === 0) {
-				pokemon.addVolatile('curse', pokemon);
+			let item = pokemon.getItem();
+			if (pokemon.isActive && !pokemon.template.isMega && !pokemon.template.isPrimal && (item.id === 'redorb' || item.id === 'blueorb') && pokemon.baseTemplate.tier !== 'Uber' && !pokemon.template.evos.length) {
+				// Primal Reversion
```
Comment (Slayer95) [inline config/formats.js]: I believe this primal reversión code may now be outdated. Have your tested it?

File: config/formats.js (hunk 1 above)
Comment (Pikachuun) [inline config/formats.js]: @Slayer95 I will correct this. For clarity's sake, you mean the mawilite case, correct?

File: config/formats.js (hunk 2 above)
Comment (Pikachuun) [inline config/formats.js]: @Slayer95 Tested it in the case of both switching in initially as well as switching back in. It worked perfectly fine in both cases.

File: config/formats.js (hunk 1 above)
Comment (Slayer95) [inline config/formats.js]: It crashes io.js@2, which we run. Yup, that case

File: config/formats.js (hunk 2 above)
Comment (Slayer95) [inline config/formats.js]: Fine

File: config/formats.js [hunk 3]
```diff
@@ -366,63 +366,147 @@ exports.Formats = [
...
+			if (item.id === 'beedrillite' || item.id === 'kangaskhanite') {
+				return ["" + item.name + " can only allowed be held by " + item.megaEvolves + "."];
+			}
+			let powerAbilities = {'Huge Power':1, 'Pure Power':1};
+			let allowedPower = false;
+			switch (item.id) {
+			case 'blazikenite':
+				if (set.ability !== 'Speed Boost') return ["" + template.species + " is not allowed to hold " + item.name + "."];
+				break;
+			case 'mawilite': case 'medichamite':
```
Comment (Pikachuun) [inline config/formats.js]: @Slayer95 I believe this is what you meant, correct?

Comment (Slayer95) [inline config/formats.js]: That works

**Pull request merged**